pkg/controller/topo/entity/reconciler.go

```diff
@@ -139,36 +172,70 @@ func (r *Reconciler) reconcileDelete(ctx context.Context, entity *v1beta1.Entity
 	nsName := types.NamespacedName{
 		Name: entity.Namespace,
 	}
-	err := r.client.Get(ctx, nsName, ns)
+	err := r.client.Get(context.TODO(), nsName, ns)
```

Comment:
I think we should be able to remove the namespace check now. That was the original solution to the same problem we're fixing by checking if the topo service exists now.